Marlin/configuration_store.cpp

```diff
@@ -824,7 +824,13 @@ void Config_ResetDefault() {
     retract_recover_feedrate_mm_s = RETRACT_RECOVER_FEEDRATE;
   #endif
 
-  volumetric_enabled = false;
+  volumetric_enabled =
+  #if ENABLED(ENABLE_VOLUMETRIC_FILAMENT_DIAMETER)
+    true
+  #else
+    false
+  #endif
+  ;
   for (uint8_t q = 0; q < COUNT(filament_size); q++)
```

Comment:
This change fails to compile.  and given the code just a few lines down does the same thing I did, I'll assume its the normal coding style.

```gcode
  endstops.enable_globally(
    #if ENABLED(ENDSTOPS_ALWAYS_ON_DEFAULT)
      (true)
    #else
      (false)
    #endif
```